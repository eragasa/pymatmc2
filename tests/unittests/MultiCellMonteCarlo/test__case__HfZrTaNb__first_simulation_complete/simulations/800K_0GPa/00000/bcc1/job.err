
Due to MODULEPATH changes, the following have been reloaded:
  1) mvapich2/2.3.2

The following have been reloaded with a version change:
  1) intel/18.0.3 => intel/19.0.5


Lmod is automatically replacing "mvapich2/2.3.2" with "intelmpi/2019.3".


Due to MODULEPATH changes, the following have been reloaded:
  1) mvapich2/2.3.2

The following have been reloaded with a version change:
  1) intel/18.0.3 => intel/19.0.5


Lmod is automatically replacing "mvapich2/2.3.2" with "intelmpi/2019.3".

