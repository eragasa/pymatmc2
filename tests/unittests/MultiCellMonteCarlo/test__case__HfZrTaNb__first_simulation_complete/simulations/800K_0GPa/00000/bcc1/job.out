
-----------------------
Resources requested:
nodes=4:ppn=40
-----------------------
Resources used:
cput=09:08:55
walltime=00:13:52
mem=8.988GB
vmem=162.579GB
-----------------------
Resource units charged (estimate):
3.697 RUs
-----------------------

-----------------------
Resources requested:
nodes=4:ppn=40
-----------------------
Resources used:
cput=00:15:47
walltime=00:00:28
mem=3.957GB
vmem=157.458GB
-----------------------
Resource units charged (estimate):
0.124 RUs
-----------------------
